GitUI/CommandsDialogs/BrowseDialog/FormBrowseMenus.cs

```diff
@@ -153,7 +153,10 @@ private static void CreateToolStripSubMenus(ToolStrip senderToolStrip, ToolStrip
                     key = toolbarItem.Name;
                 }
 
-                bool visible = LoadVisibilitySetting(key);
+                bool visible = LoadVisibilitySetting(key, IsVisibleByDefault(key));
+
+                bool IsVisibleByDefault(string buttonKey) => !buttonKey.Contains("tsbShowReflog");
```

Comment:
Although it is short and used here, this nested function should be moved behind the `return`.